README.md

```diff
@@ -7,6 +7,32 @@
 The Amazon Genomics CLI is a tool to simplify the processes of deploying the AWS infrastructure required to
 run genomics workflows in the cloud, to submit those workflows to run, and to monitor the logs and outputs of those workflows.
 
+## Sequence Bio Internal Installation
+
+Clone this repository and navigate into it:
+```bash
+git clone --branch ss-explore https://github.com/SequenceBio/amazon-genomics-cli.git
```

Comment:
Since we have merged ss-explore to the main branch. From now on, we can clone the main branch by running: 
`git clone https://github.com/SequenceBio/amazon-genomics-cli.git`

Reply:
Yeah, this looks good - I was just copying it from the Confluence doc!

Reply:
Fair enough!